src/data/gravity.graphql

```diff
@@ -1054,23 +1076,58 @@ type LotEvent {
   calculatedSellingPriceMinor: Int
   createdAt: String!
   internalID: ID!
+  isWithdrawn: Boolean
```

Comment:
`LotEvent` is [unused](https://github.com/artsy/metaphysics/blob/ff1d6f2773567f28aca75d3def1a6e8ccb226364/src/lib/stitching/gravity/schema.ts#L61) so the addition here probably doesn't matter.